ModularTegustation/tegu_mobs/apocalypse_bird.dm

```diff
@@ -398,3 +563,82 @@
 			bird_overlay.icon_state = "forest_portal2"
 
 	. += bird_overlay
+
+/datum/ai_controller/insane/enchanted
+	lines_type = /datum/ai_behavior/say_line/insanity_enchanted
+
+/datum/ai_behavior/say_line/insanity_enchanted
+	lines = list(
+				"The light...",
+				"It's so bright...",
+				"It's getting closer...",
+				"I can't look away..."
+				)
+
+/datum/ai_controller/insane/enchanted/SelectBehaviors(delta_time)
+	..()
+	if(blackboard[BB_INSANE_CURRENT_ATTACK_TARGET] != null)
+		return
+
+	current_behaviors += GET_AI_BEHAVIOR(lines_type)
+
+	var/mob/living/simple_animal/hostile/megafauna/apocalypse_bird/bird
+	for(var/mob/living/simple_animal/hostile/megafauna/apocalypse_bird/M in GLOB.mob_living_list)
+		if(!istype(M))
```

Comment:
This will never happen, mind you.
Additionally, there must be a global list of simple animals, it might be faster to search through those instead.